app/src/main/java/nodomain/freeyourgadget/gadgetbridge/GBApplication.java

```diff
@@ -197,6 +198,8 @@ public void onCreate() {
             }
             startService(new Intent(this, NotificationCollectorMonitorService.class));
         }
+
+        EmojiManager.initEmojiData(getApplicationContext());
```

Comment:
I would like to see this be done on demand, e.g. in sanitizeNotifText() so that this will not increase the startup time. Additional benefit would be that we only need to do it when really needed. 

It depends on whether initEmojiData() is callable from a background thread, though.

Reply:
That's a good idea! I just tested that and it seems it is working fine!
(at the end, this function is reading a file and parsing it, should be fine in a background thread, no?)

Reply:
Yes, that should be fine. I'm just always wary of calling "foreign" APIs from background threads, especially when it comes to initialization, loading of resources (icons) etc.